_includes/releases/v22.2/v22.2.1.md

```diff
@@ -0,0 +1,386 @@
+- Fixed an internal error that could occur when comparing a column of type void to `NULL` using `col IS NULL` or `col IS NOT NULL`. [#93680][#93680]
+- Fixed an issue where `DISTINCT ON` queries would fail with the error `"SELECT DISTINCT ON expressions must match initial ORDER BY expressions"` when the query included an `ORDER BY` clause containing `ASC NULLS LAST` or `DESC NULLS FIRST`. [#93609][#93609]
+- Previously, CockroachDB would error when receiving Geometry/Geography using binary parameters. This is now resolved. [#93685][#93685]
+- CockroachDB could previously, in rare cases, encounter an internal error when evaluating `crdb_internal.range_stats` builtin (which powers `SHOW RANGES` command among other things). The bug was introduced in v22.2.0 and is now fixed. [#93869][#93869]
+- Fixed a bug where some input to `crdb_internal.trim_tenant_prefix` would cause a node crash. [#90541][#90541]
+
+<h3 id="v22-2-1-performance-improvements">Performance improvements</h3>
+
+- The [optimizer](../v22.2/cost-based-optimizer.html) now explores plans with a single lookup join expressions in rare cases where it previously planned two lookup join expressions. [#88864][#88864]
+- Some types of queries with comparisons with constant values now execute faster. [#89554][#89554]
+- `pg_catalog.col_description` is now much faster when resolving columns for tables in the `pg_catalog`, `crdb_internal`, or `information_schema` namespaces. [#89496][#89496]
+- HTTP requests with `'Accept-encoding: gzip'` previously resulted in valid gzip-encoded, but uncompressed responses. This resulted in inefficient HTTP transfer times, as far more bytes were transferred than necessary. Those responses are now properly compressed, resulting in smaller network responses. [#89510][#89510]
+- The [optimizer](../v22.2/cost-based-optimizer.html) now does less copying of histograms while planning queries, which will reduce memory pressure. [#89957][#89957]
+- Tables in `pg_catalog` and `information_schema` (when not explicitly referenced as `"".information_schema`) may now be much faster if the current database has a small number of relations relative to the total number in the cluster. [#91054][#91054]
+- [Garbage collection](../v22.2/architecture/storage-layer.html#garbage-collection) score triggering MVCC GC run is lowered. GC score is ratio is computed from MVCC stats and uses ratio of live objects and estimated garbage age to estimate collectability of old data. Reduced score will trigger GC earlier lowering interval between runs 3 times, giving 2 times reduced peak garbage usage at the expense of 30% increase of wasteful data scanning on constantly updated data. [#92675][#92675]
+- CockroachDB in some cases now correctly incorporates the value of the `OFFSET` clause when determining the number of rows that need to be read when the `LIMIT` clause is also present. Note that there was no correctness issue here—only that extra unnecessary rows could be read. [#92840][#92840]
```

Comment:
nit: spaces surrounding hyphen: `here - only`

Reply:
This is an em dash, so there should not be spaces.